packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -153,39 +164,47 @@ const styles: ComponentStyles<ToggleClassNameContract, DesignSystem> = {
                 ...highContrastDoubleFocus,
             }),
             "&:disabled": {
+                "& $toggle_input, & $toggle_label, & $toggle_statusMessage": {
+                    [highContrastSelector]: {
+                        background: "transparent",
+                        "border-color": importantValue(HighContrastColor.disabledText),
+                        color: importantValue(HighContrastColor.disabledText),
+                        "& + span": {
+                            background: importantValue(HighContrastColor.disabledText),
+                        },
+                    },
+                },
+            },
+            "&:hover": {
                 [highContrastSelector]: {
-                    background: "GrayText !important",
-                    "border-color": "GrayText !important",
+                    background: HighContrastColor.selectedText,
+                    "border-color": HighContrastColor.selectedBackground,
                     "& + span": {
-                        background: "Background !important",
+                        background: HighContrastColor.selectedBackground,
                     },
                 },
             },
-            [highContrastSelector]: {
-                background: "Highlight",
-                "border-color": "Highlight",
-                "&:hover": {
-                    background: "HighlightText",
-                    "border-color": "Highlight",
-                },
-                "&:active": {
-                    [highContrastSelector]: {
-                        background: "Highlight",
-                        "& + span": {
-                            background: "HighlightText",
-                        },
+            "&:active": {
+                [highContrastSelector]: {
+                    background: importantValue(HighContrastColor.selectedBackground),
+                    "border-color": importantValue(HighContrastColor.selectedBackground),
+                    "& + span": {
+                        background: importantValue(HighContrastColor.selectedText),
                     },
                 },
             },
+            [highContrastSelector]: {
+                background: HighContrastColor.selectedBackground,
+                "border-color": HighContrastColor.selectedBackground,
+                "& + span": {

```

Comment:
I realize we are using "+" in a few places, but we should add more selectors so we won't need to. This is costly during rendering as the browser has to do more work to determine styles on a give element.